src/components/PopOver/index.js

```diff
@@ -135,17 +136,22 @@ class PopOver extends Component {
         ? topPosition
         : bottomPosition;
 
-    return {
-      x: Math.min(
-        Math.max(elLeft + elWidth / 2 - width / 2, containerLeft),
-        windowWidth - spaceFromEdge - width,
-        containerRight
-      ),
-      y:
-        bottomPosition + height + spaceFromEdge <=
+    const Xposition = Math.min(
+      Math.max(elLeft + elWidth / 2 - width / 2, containerLeft),
+      windowWidth - spaceFromEdge - width,
+      containerRight
+    );
+
+    const Yposition =
```

Comment:
humm normally we don't start with a capital letter, except for classes, change it to `yPosition` and `xPosition` please

Reply:
corrected